ggml/src/ggml-cpu.c

```diff
@@ -11641,79 +11641,189 @@ static void ggml_compute_forward_add_rel_pos(
     }
 }
 
-// ggml_compute_forward_rwkv_wkv
+// ggml_compute_forward_rwkv_wkv6
 
-static void ggml_compute_forward_rwkv_wkv_f32(
+static void ggml_compute_forward_rwkv_wkv6_f32(
         const struct ggml_compute_params * params,
         struct ggml_tensor * dst) {
-    const size_t T = dst->src[1]->ne[3];
-    const size_t C = dst->ne[0];
-    const size_t H = dst->src[1]->ne[2];
-    const size_t n_seqs = dst->src[5]->ne[1];
+    const int64_t T = dst->src[1]->ne[3];
```

Comment:
Similar to other operators, add any appropriate asserts here so in the future we know what are the limitations of the current implementation.

Reply:
> Similar to other operators, add any appropriate asserts here so in the future we know what are the limitations of the current implementation.

Thank you! I will add assert in sycl backend and cuda backend. As far as I know, there is no limitations in cpu backend of WKV6.